qln/lndb.go

```diff
@@ -88,8 +88,10 @@ type LitNode struct {
 	// Nodes don't have Params; their SubWallets do
 	// Param *chaincfg.Params // network parameters (testnet3, segnet, etc)
 
-	// queue for async messages to RPC user
-	UserMessageBox chan string
+	// websocket connections of users to a message channel
+	UserWsCons map[int]chan lnutil.ChatMsg
```

Comment:
(granted, UserMessageBox channel isn't a good way to do it & needs to change, BUT...)

UserWsCons in qln isn't the right way to structure this.  All the functionality in the qln package should have no idea about RPC, be it websockets or otherwise.  The litrpc package is the right place to put things that deal with websockets.  It's quite possible we'll want to make a standalone binary (maybe for phones?) which doesn't use RPC at all and has everything integrated into 1 binary.  In such a version, litrpc wouldn't exist at all, but qln should work unmodified.